.gitignore

```
@@ -146,3 +146,11 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# These folders have many files that are downloaded or temporary that do not need to go to the project repository
+kafka_2.13-2.4.0/
+temp_description_folder/
+redis-5.0.10/
```

O que acha de alterar o Kafka e o Redis para 
```suggestion
kafka_*/
temp_description_folder/
redis-*/
```
Assim, evitaria que qualquer versão futura também fosse impedida de ser enviada.

---

Seria melhor mesmo